modules/network-observability-ebpf-agent-alert.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-netobserv-dashboard-ebpf-agent-alerts_{context}"]
+= Using the eBPF agent alert
+
+When the Network Observability eBPF agent hashmap table is full, either the eBPF agent processes flows with degraded performance, or the capacity limiter is triggered. When this is the case, an alert shows `NetObservAgentFlowsDropped`. If you see this alert, consider increasing the `cacheMaxFlows` in the `FlowCollector`, as shown in the following example.
+
+[NOTE]
+====
+Increasing the `cacheMaxFlows` might increase the memory usage of the eBPF agent. 
+====
+.Prerequisite
+* You must have eBPG metrics enabled by setting the `spec.agent.type.ebpf.metrics.enable` value to `true`. For more information, see "Enabling eBPF metrics".
```

Comment:
```suggestion
* You must have eBPF metrics enabled by setting the `spec.agent.type.ebpf.metrics.enable` value to `true`. For more information, see "Enabling eBPF metrics".
```

Reply:
Apologies for the typo, the enablement was a rushed addition, and now I think I actually need to remove this prerequisite since https://github.com/netobserv/network-observability-operator/pull/649 - makes it enabled by default.

Reply:
I removed the section it references already "Enabling eBPF metrics"